dev_guide/secrets.adoc

```diff
@@ -32,23 +31,52 @@ kind: Secret
 metadata:
   name: test-secret
   namespace: my-namespace
-data: <1>
-  username: dmFsdWUtMQ0K <2>
+type: Opaque <1>
+data: <2>
+  username: dmFsdWUtMQ0K <3>
   password: dmFsdWUtMg0KDQo=
-stringData: <3>
+stringData: <4>
   hostname: myapp.mydomain.com <4>
 ----
-<1> The allowable format for the keys in the `data` field must meet the
+<1> Indicates the xref:types-of-secrets[structure of the secret’s key names and values].
+<2> The allowable format for the keys in the `data` field must meet the
 guidelines in the *DNS_SUBDOMAIN* value in
```

Comment:
fix the link to be https://github.com/kubernetes/kubernetes/blob/v1.0.0/docs/design/identifiers.md